### PR TITLE
Updated Datek APEX and added Datek HAN adapter

### PR DESCRIPTION
**Pull request by TheStigh**
Updated Datek APEX and added Datek HAN for firmware 0.2, next firmware for HAN will include temperature. New OTA file already mergen by Koen.

### CODE REVIEW

File: devices.js [hunk 1]
```diff
@@ -17260,16 +17260,46 @@ const devices = [
...
+            await reporting.rmsCurrent(endpoint);
+            await reporting.activePower(endpoint);
+            await reporting.temperature(endpoint);
+        },
+        exposes: [e.power(), e.current(), e.voltage(), e.switch(), e.temperature()],
+    },
+
+    {
+        zigbeeModel: ['HAN'],
+        model: 'Datek HAN',
```
Comment (Koenkk) [inline devices.js:17283]: Is there any model written on the device? (one where you can find the device with via Google?)

**Comment by Koenkk** [comment]
Note to self: after this merge https://github.com/Koenkk/zigbee-herdsman-converters/pull/2423

File: devices.js (hunk 1 above)
Comment (TheStigh) [reply devices.js:17283]: Nop, I did ask the manufacturer but the model name are DATEK HAN. That is why I in the device information have provided URL to the product page.

Comment (TheStigh) [reply devices.js:17283]: @Koenkk Can this be merged please (you said within hours) so I can get the OTA available to upgrade the HAN adapter, will be more available + fixing stability issues in this old fw.

**Comment by Koenkk** [comment]
Thanks!

**Pull request merged**